contracts/test/permission/auth/auth.ts

```diff
@@ -1,7 +1,5 @@
 import {expect} from 'chai';
 
-// TODO
-
 describe.skip('auth', async () => {
```

Comment:
Can you explain via TODO in code, or via PR description why:
- this test is skiped?
- in what task it will be addressed?

Reply:
All of them are skipped because they are non-existing. All skipped tests contain `expect(false).to.be.true`.

I thought it is better to put at least `// TODO` and/or test sekeletons there instead of nothing.
Before we release a release candidate, all of them must be gone, ofc.